pages/resultset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.base import Base
+
+
+class ResultsetPage(Base):
+
+    _job_log_locator = (By.CSS_SELECTOR, '.logviewer-icon')
+    _job_resultpane_status = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span:nth-child(2)')
+    _log_viewer_locator = (By.CSS_SELECTOR, 'logviewer-icon')
+    _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
+    _result_status_locator = (By.ID, 'result-status-pane')
```

Comment:
Good stuff! I had to think about this a bit also. Assuming `locator` is a convention, here's a tweak if you think makes sense. We use "job details" to generally describe the entire panel which opens when a job is selected. Note I have slightly re-ordered them:

```
_job_details_logviewer_icon_locator = (By.CSS_SELECTOR, '.logviewer-icon')
_job_details_logviewer_btn_locator = (By.CSS_SELECTOR, 'logviewer-icon')
_job_details_result_status_pane = (By.ID, 'result-status-pane')
_job_details_result_status = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span:nth-child(2)')
_resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
```

So we could generally use these terms I think, for the main UI:
\_navbar (main navbar)
\_navbar_quickfilter
\_resultset
\_resultset_revisions (left side)
\_resultset_jobs (right side)
\_resultset_bar (the per resultset bar containing Date/Author/SHA and action menu dropdown at the right side)
\_job_details
\_job_details_navbar
\_job_details_actionmenu (the left side buttons with log icon, rawlog icon, pin icon, retrigger icon, etc)
\_job_details_tab_\[name\] (the tabs "job details", "failure summary", etc)
\_pinboard
\_logviewer
\_logviewer_header
\_logviewer_failuresteps
\_logviewer_body
\_help

If the above is too granular, feel free to turf what isn't needed :)